packages/bpk-component-input/README.md

```diff
@@ -47,6 +47,13 @@ export default () => (
 | onClear          | func                       | if clearable={true} | null                     |
 | valid            | bool                       | false               | null                     |
 
+## Theme Props
+
+* `inputValidIconColor`
+* `inputInvalidIconColor`
+* `inputInvalidBorderColor`
+* `inputInvalidBackgroundColor`
```

Comment:
Looks like a few of these lines are left in from when I originally was trying to theme the icon colour, should probably be removed.

Reply:
Updated this